.prettierignore

```diff
@@ -0,0 +1,2 @@
+package.json
```

Comment:
why?

Reply:
To cater for VSCode users that use`prettier on save` plugins. PHPStorm too I suppose.

Reply:
Probably a better alt is to modify `overrides` and set spacing to 2? Helps apply it everywhere and not just this codebase.

Reply:
Okay, looks like overrides are doing its job. Oops didn't check for that.